roles/sat6repro/tasks/main.yml

```diff
@@ -103,6 +105,9 @@
   file: src=/opt/rh/ruby193/root/usr/share/gems/gems/katello-2.2.0.93/public/assets/bastion_katello dest=/usr/share/foreman/public/assets/bastion_katello
   when: satelliteversion == 6.1
 
+- include: reset_pulp_data.yml
```

Comment:
If you clone w/o pulp_data file, the entries for rpms in repositories will still be in the pulp db. So pulp will think everything is synced, and wont re-sync new content. Also, the package counts will be wrong.

The steps in this yml file clean the pulp database, returning everything to an "empty, unsynced" state.